packages/stainless/src/stl.ts

```diff
@@ -190,74 +256,146 @@ export class StlError extends Error {
   }
 }
 
+/** When thrown in an endpoint handler, responds with HTTP status code 400. */
 export class BadRequestError extends StlError {
+  /**
+   * @param response  optional data included in the body of the response JSON.
+   */
```

Comment:
I don't think these comments add that much value… would be fine at the base class though…

Reply:
I dunno, superclass only isn't particularly helpful on hover, which is relevant imo.

Reply:
right – point is moot iff we get rid of the per-subclass constructors, but yeah we'd want to do that in a separate PR